{"uuid": "60bdf190-58d9-4f78-8f45-71b4753f3204", "children": ["5143575d-9353-4531-afd4-9f635cd7c0a0"], "befores": [{"name": "setup_driver_class", "status": "passed", "start": 1755764156643, "stop": 1755764158415}], "afters": [{"name": "setup_driver_class::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\eruba\\PycharmProjects\\PythonAutomation\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\eruba\\PycharmProjects\\PythonAutomation\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\eruba\\OneDrive\\מסמכים\\GitHub\\shichor-automation\\tests\\conftest.py\", line 37, in setup_driver_class\n    driver.quit()\n    ~~~~~~~~~~~^^\n  File \"C:\\Users\\eruba\\PycharmProjects\\PythonAutomation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 224, in quit\n    self.service.stop()\n    ~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\eruba\\PycharmProjects\\PythonAutomation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 152, in stop\n    self.send_remote_shutdown_command()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\eruba\\PycharmProjects\\PythonAutomation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 137, in send_remote_shutdown_command\n    if not self.is_connectable():\n           ~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\eruba\\PycharmProjects\\PythonAutomation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 126, in is_connectable\n    return utils.is_connectable(self.port)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\eruba\\PycharmProjects\\PythonAutomation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 99, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n  File \"C:\\Program Files\\Python313\\Lib\\socket.py\", line 856, in create_connection\n    exceptions.clear()  # raise only the last error\n    ~~~~~~~~~~~~~~~~^^\n"}, "start": 1755764159575, "stop": 1755764160636}], "start": 1755764156643, "stop": 1755764160636}